src/filter.py

```diff
@@ -126,3 +127,68 @@ def release_type(results):
         for result in results
         if get_setting('include_release_' + result["release_type"], bool)
     ]
+
+
+def tv_season_episode(results, season, season_name, episode, global_ep, ep_year, season_year=0, start_year=0):
+    # Function for sorting large amount of not accurate torrents. Checks year season and episodes.
+    # The filtering shouldn't be strict. I'm trying not to lose suitable torrents.
+    filtered = []
+    for res in results:
+        name = res["name"].lower()
+        # Remove resolution
+        name = re.sub(r"\d+p", '', name)
+        log.debug(f"torrent: {name}")
+
+        if season_name and season_name in name:
+            filtered.append(res)
+            continue
+
+        year_pattern = r"(?P<from>(?:19|20)\d{2})(?:\s*-\s*(?P<to>[12]\d{3}))?"
+        y = re.search(year_pattern, name)
+        if not y:
+            log.debug("No year")
+            continue
+        y_from = int(y.group("from") or "99999")
+        y_to = int(y.group("to") or "-1")
+        if (start_year != y_from and ep_year != y_from and season_year != y_from and
+                (y_from > season_year or season_year > y_to)):
+            log.debug(f"Not suitable year: {ep_year or 'none'} || {season_year or 'none'} || {start_year or 'none'}")
+            continue
+        # Remove the year from the text
+        name_no_year = re.sub(year_pattern, '', name)
+
+        if f"s{season}e{episode}" in name_no_year:
+            filtered.append(res)
+            continue
+
+        season_pattern = r"\W(?P<s_flag>s|season|сезон|tv-?|тв-?)[\s\(\[\{]*(?P<from>\d+)(?:\s*-\s*(?P<to>\d+))?"
```

Comment:
I appreciate you're adding your own language, but we'd have to add other languages here too. Often these languages wouldn't be something a user wants. I'm wondering what the right solution would be here. Maybe an additional config option?

Reply:
yes you are right. I thought it was "not good enough". But code became better and i pointed place for langspecific string. 
I thing the best way will be to use existing method with https://github.com/fugkco/script.elementum.jackett/tree/master/resources/language
+ adding config property like: `Extra language for torrent filtering`
I was just a little bit tired to do this. And not sure if it's the best method.